File: rest/src/main/java/io/sparkled/viewmodel/playlist/sequence/PlaylistSequenceViewModelConverter.java
package io.sparkled.viewmodel.playlist.sequence;

import io.sparkled.model.entity.PlaylistSequence;
import io.sparkled.viewmodel.ModelConverter;
import io.sparkled.viewmodel.ViewModelConverter;

public abstract class PlaylistSequenceViewModelConverter implements ModelConverter<PlaylistSequence, PlaylistSequenceViewModel>,
        ViewModelConverter<PlaylistSequenceViewModel, PlaylistSequence> {
}

File: rest/src/main/kotlin/io/sparkled/viewmodel/playlist/sequence/PlaylistSequenceViewModelConverter.kt
package io.sparkled.viewmodel.playlist.sequence

import io.sparkled.model.entity.PlaylistSequence
import io.sparkled.viewmodel.ModelConverter
import io.sparkled.viewmodel.ViewModelConverter

abstract class PlaylistSequenceViewModelConverter : ModelConverter<PlaylistSequence, PlaylistSequenceViewModel>, ViewModelConverter<PlaylistSequenceViewModel, PlaylistSequence>
